### ISSUE TITLE
Test Data Structures to ensure they are compatible

### ISSUE THREAD

**Issue by paulvxx** [cs481-ekh/s24-meshers#30]
References #16

- [x] The data structures used for 2d Points and systems of linear equations should be tested for accessibility and easy manipulation
- [x] Ideally, find a C++ function to use within python to test these structures and ensure they work as expected

**Comment by WorkGitAccount** [comment]
Python libraries (sciPy numPy) for using linear equations linked and working. 

This task is a basic verification and not a thorough test.  Want to verify that the C++ cyPackage files linked from Task #25 are working as expected. Walked through the C++ coded and verified some dependencies. Found the C++ files in /util/cyCodeBase that are actually needed in the sample elimination phase. 

Considered removing the unnecessary files in /util/cyCodeBase for simplicity while some files can be kept. Like some graphics files may be useful in plotting and graphing but aren't strictly necessary.  For example the C++ files referencing WebGL and GLSL. These languages are useful for graphics and utilize parallelism on the GPU.  The PointCloud class is also setup for parallelism but it is not currently being utilized. 

Could revisit these parallelism possibilities 

[t:45]
